inc/thorvg_capi.h

```diff
@@ -161,7 +161,9 @@ TVG_EXPORT Tvg_Gradient* tvg_radial_gradient_new();
 TVG_EXPORT Tvg_Result tvg_linear_gradient_set(Tvg_Gradient* grad, float x1, float y1, float x2, float y2);
 TVG_EXPORT Tvg_Result tvg_radial_gradient_set(Tvg_Gradient* grad, float cx, float cy, float radius);
```

Comment:
Please add getters for linear_gradient_set and radial_gradient_set in this commit.